kwasm-lemmas.md

```diff
@@ -24,6 +24,17 @@ Basic arithmetic
       requires 0 <=Int X
        andBool X  <Int N
       [simplification]
+
+    rule (X *Int N +Int Y) modInt N => Y modInt N
+      [simplification]
+    rule (Y +Int X *Int N) modInt N => Y modInt N
+      [simplification]
```

Comment:
can we have `[simplification]` on the same line as `rule` so they line up nicer?